.circleci/config.yml

```diff
@@ -39,193 +35,108 @@ jobs:
+      - run:
+          name: Authenticate to Docker Hub
+          command: echo ${DOCKERHUB_PASS} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin
+      - run:
+          name: Load Archived Docker image
           command: docker load -i /tmp/workspace/image.tar
       - run:
           name: Publish Docker Image to Docker Hub as Sandbox
-          command: |
-            echo "${DATESTAMP}"
-            echo "${DOCKERHUB_PASS}" | docker login -u "${DOCKERHUB_USERNAME}" --password-stdin
-            docker push ${IMAGE_NAME}:sandbox-${DATESTAMP}
-            docker run -e IMAGE_TO_SCAN=${IMAGE_NAME}:sandbox-${DATESTAMP} -e SYSDIG_SECURE_TOKEN=$SYSDIG_SECURE_TOKEN logdna/sysdigsecure
-  docker-approved:
-    executor: docker-publisher
-    steps:
-      - attach_workspace:
-          at: /tmp/workspace
-      - run: cat /tmp/workspace/env-vars >> $BASH_ENV
-  docker-upload:
+          command: docker push ${IMAGE_NAME}:sandbox-${DATESTAMP}
+      - run:
+          name: Scan Sandbox Image
+          command: docker run -e IMAGE_TO_SCAN=${IMAGE_NAME}:sandbox-${DATESTAMP} -e SYSDIG_SECURE_TOKEN=${SYSDIG_SECURE_TOKEN} logdna/sysdigsecure
+  docker-hub-upload:
     executor: docker-publisher
     steps:
       - attach_workspace:
           at: /tmp/workspace
-      - run: cat /tmp/workspace/env-vars >> $BASH_ENV
       - setup_remote_docker
       - run:
-          name: Load archived Docker image
+          name: Set Environment Variables
+          command: cat /tmp/workspace/env-vars >> ${BASH_ENV} && source ${BASH_ENV}
+      - run:
+          name: Authenticate to Docker Hub
+          command: echo ${DOCKERHUB_PASS} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin
+      - run:
+          name: Load Archived Docker image
           command: docker load -i /tmp/workspace/image.tar
       - run:
-          name: Publish Docker Image to Docker Hub
+          name: Tag and Push Docker Image to Docker Hub
           command: |
-            echo "${DOCKERHUB_PASS}" | docker login -u "${DOCKERHUB_USERNAME}" --password-stdin
-            TAG=${CIRCLE_TAG}
-            D_TAG="$(wget -q https://registry.hub.docker.com/v1/repositories/logdna/logdna-agent/tags -O - | jq -r '.[] | .name' | awk '!/-/ && /\./ { print $1}' | grep ${MAJOR_VERSION} | tail -1)"
-            if [ $(echo $TAG | wc -m) -lt 3 ]; then
-              IMAGE_TAG=${D_TAG}
-            else
-              IMAGE_TAG=${TAG}
-            fi
-            echo "${IMAGE_TAG}"
-            docker tag ${IMAGE_NAME}:sandbox-${DATESTAMP} ${IMAGE_NAME}:latest
-            docker tag ${IMAGE_NAME}:latest ${IMAGE_NAME}:${IMAGE_TAG}
-            docker tag ${IMAGE_NAME}:latest ${IMAGE_NAME}:${IMAGE_TAG}-${DATESTAMP}
-            docker push ${IMAGE_NAME}:${IMAGE_TAG}
-            docker push ${IMAGE_NAME}:${IMAGE_TAG}-${DATESTAMP}
-            docker push ${IMAGE_NAME}:latest
-  ibm-registry:
+            TAGS=( latest ${IMAGE_TAG} ${IMAGE_TAG}-${DATESTAMP} )
```

Comment:
I thought we were tagging `${MAJOR_VERSION}` before. I don't see it on the old version, but I do still think we should have one. just a thought.

Reply:
Yes, first removed since wasn't seeing any point in keeping - now I see - added back